src/env/env.rs

```diff
@@ -132,6 +141,20 @@ pub fn uumain(args: Vec<String>) -> i32 {
                         Some(s) => opts.unsets.push(s.to_owned()),
                     }
                 }
+                prefix if prefix.starts_with("--split-string") => {
+                    let length = "--split-string".len();
+                    if prefix.len() == length { // when used like "env --split-string 'foo bar'"
+                        let string = iter.next();
+                        match string {
+                            None => eprintln!("{}: this option requires an argument: {}", NAME, opt),
+                            Some(s) => opts.program.append(&mut split_string(s)) ,
```

Comment:
If you change `split_string()` to return an iterator, this can just be `opts.program.extend(split_string(s))`.